experiment_handler/experiment_handler.py

```diff
@@ -111,7 +112,7 @@ def retrieve_experiment():
 
     list_experiments = []
     for class_name, class_obj in experiment_classes.items():
-        if experiment_proto_class in inspect.getmro(class_name):  # an experiment
```

Comment:
I assume this didn't work with just the name, and you needed to pass the actual class object to getmro? The documentation doesn't say.